lib/widgets/message_list.dart

```diff
@@ -392,12 +393,18 @@ class MarkAsReadWidget extends StatelessWidget {
           // TODO(#368): this should pull from stream color
           color: Colors.transparent,
           child: Padding(
-            padding: const EdgeInsets.all(10),
+            // vertical padding adjusted for tap target height (48px) of containing button
+            padding: const EdgeInsets.symmetric(horizontal: 10, vertical: 10 - ((48 - 38) / 2)),
             child: FilledButton.icon(
               style: FilledButton.styleFrom(
-                padding: const EdgeInsets.all(10),
-                textStyle: const TextStyle(fontSize: 18, fontWeight: FontWeight.w200),
-                shape: RoundedRectangleBorder(borderRadius: BorderRadius.circular(5)),
+                backgroundColor: _UnreadMarker.color,
```

Comment:
msglist [nfc]: Factor out color for _UnreadMarker and MarkAsReadWidget

This change isn't NFC, because it changes the color of MarkAsReadWidget.

Could have an NFC commit without this line, just pulling out the static `_UnreadMarker.color`, and then a second commit to use it here. Or this commit is fine as is, it just isn't NFC (and what it's doing isn't "factoring out" from MarkAsReadWidget, because this color wasn't previously there) — a good commit message would be `msglist: Color MarkAsReadWidget the same as _UnreadMarker`.